styles.css

```diff
@@ -66,4 +67,41 @@ html {
 #github_logo {
     margin-left: 1000px;
     opacity: 0.5;
+}
+
+#bolgeler {
+    display: none;
+    border-radius: 2em;
+    width: 300px;
+    height: 200px;
+    padding-top: 20px;
+    padding-bottom: 40px;
+    padding-left: 40px;
+    background-color: #EFAE88;
+    font-family: "Roboto", sans-serif;
+    font-size: 80%;
+    margin-top: 25px;
```

Comment:
- Kutucuğun, üstündeki yazıya bitişmesini engellemek için 25 pixel boşluk açıyoruz.